README.md

```diff
@@ -67,11 +71,17 @@ const options = {
   },
 
 ```
+Other properties are the following.
+
+####packages property
+
 The `packages` property specifies which accelerator packs should be included in the application.  If using a bundler like [`webpack`](https://webpack.github.io/) or [`Browserify`](http://browserify.org/), you'll need to install the additional packages using `npm`.  Otherwise `Core` will look for them in global scope.
 
 ```javascript
   packages: ['textChat', 'screenSharing', 'annotation', 'archiving'],
 ```
+
+#### streamContainers property
```

Comment:
We talk about multiple options properties here, not just `streamContainers`.  Maybe we should change the subheader to `Containers`.